### PR TITLE
Support for Indicator Command Class

### PR DESCRIPTION
**Pull request by paixaop**
Chris created a new pull request with just the Indicator Class changes so you can merge it.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommandClass.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+ * This can be used to identify a device or use the indicator for special purposes.
+ * Example is the Evolve LCD panel that uses the Indicator class to toggle the labels 
+ * displayed on the LCD. The Indicator class is also used to sync multiple panels' labels
+ * 
+ * @author Pedro Paixao
+ * @since 1.8.0
+ */
+
+@XStreamAlias("indicatorCommandClass")
+public class ZWaveIndicatorCommandClass extends ZWaveCommandClass implements ZWaveIndicatorCommands {
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommandClass.java]: Please remove the ZWaveIndicatorCommands

File: bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommands.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveIndicatorCommands.java]: I think we discussed earlier that this class is not needed? If so, please remove it - it doesn't appear to be used and it's completely empty... (or am I missing something?)

**Comment by paixaop** [comment]
Removed the ZWaveIndicatorCommands class

**Comment by cdjackson** [comment]
Thanks.

**Pull request merged**